rita/src/rita_common/http_client/mod.rs

```diff
@@ -73,7 +73,12 @@ impl Handler<Hello> for HTTPSyncExecutor {
 
     fn handle(&mut self, msg: Hello, _: &mut Self::Context) -> Self::Result {
         info!("sending {:?}", msg);
-        let my_id = serde_json::to_string(&msg.my_id)?;
+
+        // TODO: REMOVE IN ALPHA 5
```

Comment:
@kingoflolz @jkilpatr Where would be the best place to note this? Our Trello usage somewhat died of natural causes, maybe we need a different approach? It would be nice to have a fix for this actually scheduled somewhere

Reply:
https://forum.altheamesh.com/t/tunnel-manager-change/38

We put it there but I'll admit trello is nice for this sort of stuff. What are your thoughts?

Reply:
Awesome! Yeah, I don't think we will find anything great anytime soon, each one of us probably has their favourite way of juggling tasks. Maybe some good Trello <-> Github integration would help because all the Trello goodness is already in place but you need to manually go and mark the cards done when coding work is done, which probably is the root cause of us abandoning Trello. That and the size of our team (and project) which IMHO still isn't big enough to be seriously threatened by forgotten hacks.

Reply:
Another option is to move all code-related tasks to GitHub issues altogether and only manage theory/design problems on Trello

Reply:
I'm not opposed to issues, we just have to keep up with them. I've liked the forum so far in that a post makes reporting easier.